### DISCUSSION TITLE
Bridging of HBTC from ETH to SOL not received

### DISCUSSION THREAD

**Discussion by quentinadam** [wormhole-foundation/wormhole#455] [Problems / bugs] [answered]
Hello,

I tried to bridge 0.001 HBTC from ETH to SOL.
Here is the ETH transaction: https://etherscan.io/tx/0xbe7dbeb4aa6c2eee2a0f4551cbf244347e044232cbaa9aad1e4502533f686d11.
It has been nearly a day now and I still haven't received the HBTC on Solana.
Can you please check ?

**Comment by leoluk** [comment, marked as answer]
Looks like the token account is missing: https://explorer.solana.com/tx/2Skq8LqfCVG3hCyYeHG2o89T6kvQM3wDVWJHuxbq2TPPw6gMsRyWj2PsvX5yEDRszcgCwtaKYttcLHTWgMegu2Mt

Please follow yamijuan's advice here to create the missing account: https://github.com/solana-labs/oyster/issues/196#issuecomment-909535731. Let me know once the account exists.

**Reply by quentinadam** [reply]
Thanks for your answer, but this is odd, because I did already have a token account for wHBTC (mint: https://explorer.solana.com/address/8pBc4v9GAwCBNWPB5XKA93APexMGAS4qMr37vNke9Ref) : https://explorer.solana.com/address/Emzws9C56LUk2u4oovwsJBLA4UL5mzG8fFnHoFY7kYBL.

I have now put a tiny amount of wHBTC on the token account. Will that change anything ?

**Reply by leoluk** [reply]
You need the specific derived token account Wormhole wants to create. Following the steps above should create it.

**Reply by quentinadam** [reply]
Ok, so I went on the web GUI to try to send a small amount (0.001073) of HBTC from ETH to SOL.
The ETH transaction has been confirmed correctly (https://etherscan.io/tx/0xff0c573645620b681e04b61743a2a03a2ac654525ac8088f740c226c2834ad0e) and I did receive 0.001073 wHBTC on Solana (https://explorer.solana.com/tx/3rjzMevLgYNUA7JNzRKaKHPXFyqL1ee2SBgPr3XFfUxPC22Vo6wJXjZicEkUdPdagXL2qwg5b4P4TFYAEp7ZzynD).

**Reply by quentinadam** [reply]
Ok I think believe I found out what has been going wrong.
The original eth transaction (https://etherscan.io/tx/0xbe7dbeb4aa6c2eee2a0f4551cbf244347e044232cbaa9aad1e4502533f686d11 ) where I tried to bridge 0.001 HBTC was created with value "abf0f6a6cd9b06c5759ff36275c484ae6b67e25020fe647e1a6aa1ac178d901b" as third parameter (recipient). This corresponds to my main Solana address and not the the associated token account address for wHBTC. The value should have been "ccaf015cf769b55dd38b4d2c6252635c9de0ef2c131d6b2b87ce8aa19b065fab".
Are the tokens lost then ?

**Reply by leoluk** [reply]
Weird, how did this happen? Did you create the transaction manually?

> Are the tokens lost then ?

I think so :(

**Reply by quentinadam** [reply]
Yes, I did create the transaction manually, because I am working on some automation.
Ok then they are lost, it's only $50 (less than an ethereum transaction cost).
As a more broader question, this was a "v1 Wormhole transaction". Will this continue to be supported ?

**Reply by leoluk** [reply]
> Will this continue to be supported ?

Kinda sorta... the v1 network will be up for as long as the guardians want to keep the lights on, but the ecosystem is migrating to v2 tokens. There won't be any new features or chains coming to v1, the protocols like Saber plan to migrate pools and incentives to v2 tokens. The new UI will ask users to swap their v1 tokens to v2.
